behat/features/bootstrap/FeatureContext.php

```diff
@@ -211,4 +231,160 @@ public function nodeOfTypeShouldBeDeleted($title, $type) {
+    $wrapper = entity_metadata_wrapper('message', $message);
+    $wrapper->field_ci_build_status->set(strtolower($status));
+    $wrapper->save();
+
+    $this->iSetStatusForCiBuildItemsTimes($status, $ci_build, $times - 1);
+  }
+
+  /**
+   * @Then I should see status for CI build :ci_build equal to :status
+   */
+  public function iShouldSeeStatusForCiBuildEqualTo($ci_build, $status) {
+    $status = (strtolower($status) == 'ok') ? NULL : strtolower($status);
+
+    $query = new EntityFieldQuery();
+    $entity = $query->entityCondition('entity_type', 'node')
+      ->propertyCondition('type', 'ci_build')
+      ->propertyCondition('title', $ci_build)
+      ->range(0, 1)
+      ->execute();
+
+    $node = node_load(array_keys($entity['node'])[0]);
+    $wrapper = entity_metadata_wrapper('node', $node);
+
+    $ci_build_status = $wrapper->field_ci_build_incident_status->value();
+    if ($ci_build_status != $status) {
+      $params = [
+        '@title' => $ci_build,
+        '@ci_build_status' => $ci_build_status,
+        '@status' => $status
+      ];
+      throw new \Exception(format_string("CI build @ci_build have status @ci_build_status instead of @status", $params));
+    }
+
+  }
+
+  /**
+   * @Then I should see failed count for CI build :ci_build equal to :count
+   */
+  public function iShouldSeeFailedCountForCiBuildEqualTo($ci_build, $count) {
+    $query = new EntityFieldQuery();
+    $entity = $query->entityCondition('entity_type', 'node')
+      ->propertyCondition('title', $ci_build)
+      ->propertyCondition('type', 'ci_build')
+      ->propertyOrderBy('vid', 'DESC')
+      ->range(0, 1)
+      ->execute();
+
+    $ci_build = node_load(array_keys($entity['node'])[0]);
+    $wrapper = entity_metadata_wrapper('node', $ci_build);
+    $failed_count = $wrapper->field_ci_build_failed_count->value();
+    if ($failed_count != $count) {
+      $params = [
+        '@title' => $ci_build,
+        '@failed_count' => $failed_count,
+        '@count' => $count
+      ];
+      throw new \Exception(format_string('CI build @title have failed count equal to @failed_count instead of @count', $params));
+    }
+  }
+
+  /**
+   * @Then I should see incident with status :status for CI build :ci_build
+   */
+  public function iShouldSeeIncidentForCiBuild($status, $ci_build) {
+    // Get ID of CI build.
+    $query = new EntityFieldQuery();
+    $entity = $query->entityCondition('entity_type', 'node')
+      ->propertyCondition('title', $ci_build)
+      ->propertyCondition('type', 'ci_build')
+      ->propertyOrderBy('vid', 'DESC')
+      ->range(0, 1)
+      ->execute();
+
+    $ci_build_id = array_keys($entity['node'])[0];
+    if (!$ci_build_id) {
+      $params = [
+        '@title' => $ci_build
+      ];
+      throw new \Exception(format_string('Failed to get ID of CI build @title', $params));
+    }
+
+    // Get incident.
+    $query = new EntityFieldQuery();
+    $entity = $query->entityCondition('entity_type', 'node')
```

Comment:
I have problem with this query. In `devel/php` it is work but here I receive empty result.